script/audit_deps.py

```diff
@@ -22,6 +22,16 @@
     os.path.join('vendor', 'brave-extension', 'node_modules'),
 ]
 
+# Tag @sec-team before adding any advisory to this list
+# Ignore these rust advisories
+IGNORED_NPM_ADVISORIES = [
```

Comment:
That should be `IGNORED_CARGO_ADVISORIES` or `IGNORED_RUSTSEC_ADVISORIES` ideally.

Reply:
fixed!